assets/stylesheets/main.css

```diff
@@ -212,6 +214,38 @@ a:hover#join {
   border-bottom: 1px solid aliceblue;
 }
 
+/*
+    =================================================
+    All sections
+    =================================================
+*/
+
+/* All sections EXCEPT first, b/c it uses padding instead. This approach
+is from https://css-tricks.com/hash-tag-links-padding/ */
+
+#introduction::before,
+#section3::before,
+#section3a::before,
+#section3b::before,
+#section3c::before,
+#attribution::before,
+#section4::before,
+#current-projects::before,
+#section6::before,
+#section7::before,
+#section8::before {
+    display: block;
+    content: " ";
+    margin-top: -8vh;
+    height: 8vh;
+    visibility: hidden;
+    pointer-events: none;
+}
+
+/* To use z-index, all subsections must have a position other than 'static'  */
+.setSectionPosition {
```

Comment:
In order for z-index to work, sections must have a value for the "display" property other than "static". This class sets the value to "relative".